tests/test_base.py

```diff
@@ -17,7 +17,7 @@
 
 oven.engine.execute(TABLEDEF)
 oven.engine.execute(
-    "insert into foo values ('hi', 'there', 5, '2000-01-01', '2000-01-01 12:15:00'), ('hi', 'fred', 10, '2015-05-15', '2013-10-15 05:20:10')"  # noqa: E501
+    "insert into foo values ('hi', 'there', 5, '2005-01-01', '2005-01-01 12:15:00'), ('hi', 'fred', 10, '2015-05-15', '2013-10-15 05:20:10')"
```

Comment:
One of our tests looks back 20 years and performs calculations. This caused a failure on Jan 1 2020.